packages/ChangesReloaded-Core.package/CRChangeListItem.class/instance/hash.st

```diff
@@ -0,0 +1,9 @@
+comparing
+hash
```

Comment:
Do we need that? I've seen that the linter didn't like the `=` without the `hash`, but why tho?

Reply:
If I knew...

Reply:
Hang on, so hash should deliver the same value if the obejcts differ only at the Object Identity? This is not the case right now

Reply:
no, hash should only deliver the same value for the exact same objects (with the same object identity) because this is what hashes are there for... right?

Reply:
If two objects equal each other (`=`), then their hashes are required to also be equal.
If two hashes are equal, that doesn't necessarily mean that objects are also equal.

Object identity is completely separate from both `=` and `hash` and has it's own respective concepts, `==` and `identityHash`.

Reply:
But `hash` of the ChangeRecord uses the identityHash to create its hash. How to deal with that?

Reply:
Well, by default, to `Object`s are not `=`, so it's fine that their `hash` is not the same.
As soon as you override `=`, you should also override `hash` though.